ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PostgresStorageBackend.scala

```diff
@@ -257,7 +257,7 @@ private[backend] object PostgresStorageBackend
     val pgSimpleDataSource = new PGSimpleDataSource()
     pgSimpleDataSource.setUrl(jdbcUrl)
 
-    Using.resource(pgSimpleDataSource.getConnection())(checkCompatibility)
+    Try(Using.resource(pgSimpleDataSource.getConnection())(checkCompatibility))
```

Comment:
This silently drops any errors, wouldn't it be better to use `com.daml.timer.RetryStrategy.exponentialBackoff` or similar to retry at least a few times?

Reply:
I don't think it is a good idea to retry here, see explanation https://github.com/digital-asset/daml/pull/10758#issuecomment-911938140
And yes this will silently drop errors. But still it will run multiple times as we instantiate other datasources later, when connectivity should be already there. Not ideal I know.
The way I see it either we do something quickly if this is pushing (like this) and then we do the proper thing, or we just do the proper thing.